### ISSUE TITLE
http4k-format-jackson incompatible with jackson-module-kotlin 2.10.1

### ISSUE THREAD

**Issue by mengelbrecht** [http4k/http4k#320]
When using `http4k-format-jackson` and `jackson-module-kotlin` 2.10.1 the `Jackson` object causes a `NoSuchMethod` error:
```log
Exception in thread "main" java.lang.NoSuchMethodError: 'void com.fasterxml.jackson.module.kotlin.KotlinModule.<init>(int, boolean, boolean, int, kotlin.jvm.internal.DefaultConstructorMarker)'
	at org.http4k.format.Jackson.<init>(Jackson.kt:12)
	at org.http4k.format.Jackson.<clinit>(Jackson.kt:12)
```

Sample code:
```kotlin
import org.http4k.format.Jackson

fun main(args: Array<String>) {
    Jackson.toString()
}
```

This issue happens particularly when using the Spring-Boot 2.2.2 dependency management which includes `jackson-module-kotlin` 2.10.1.

**Comment by daviddenton** [comment]
We do need to upgrade to 2.10.1, but are waiting to see what happens with this issue that we raised on it: 

https://github.com/FasterXML/jackson-module-kotlin/issues/270

What happens if you force the version of the Kotlin module down to 2.10.0?

**Comment by mengelbrecht** [comment]
Thanks for your quick response. Manually setting the version to 2.10.0 works.

**Comment by daviddenton** [comment]
Update: #270 is now fixed in Jackson-kotlin-module, so we're just waiting on a release

**Comment by daviddenton** [comment]
This will be released in v3.207.0 (should be in the next day or so)
